apps/dashboard/app/models/project.rb

```diff
@@ -302,6 +302,8 @@ def update_attrs(attributes)
   def make_dir
     project_dataroot.mkpath         unless project_dataroot.exist?
     configuration_directory.mkpath  unless configuration_directory.exist?
+    Workflow.workflow_dir(project_dataroot)
+    JobLogger::JobLoggerHelper.log_file(project_dataroot)
```

Comment:
I'm not in love with the fact that you're calling these expressly for the side affects and not the return value.

The `log_file` especially should fix itself whenever `upsert_job!` or similar is called.

Can't speak to the `workflow_dir` but it seems like it should follow the same pattern of `mkpath unless <it> exist?`

There's got to be a better way here.